services/core-web/src/components/common/GenerateDocumentFormField.js

```diff
@@ -72,7 +72,6 @@ const FIELDS = {
   MULTI_SELECT: GenerateDocumentFormField,
   RADIO: GenerateDocumentFormField,
   GROUPED_SELECT: GenerateDocumentFormField,
-  MINE_SELECT: GenerateDocumentFormField,
```

Comment:
This file is 🍌 🍌 🍌 
There is one spot where we store the form structure... in the database.
So I ran a few queries like `SELECT * FROM document_template WHERE form_spec_json LIKE '%LABEL%'; to see if I could delete anything`.
Highly recommend updating these instances in the future if you come across them because of how difficult it is to maintain.